### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by annapasca**
In this dev version mainly there are

- new tool to read/write hdf5 files (ephypype/import_hdf5.py, ephypype/nodes/import_data.py)

- reorganization of the computation of the inverse solution. In particular I added the possibility to compute and save the inverse solution for all points of the source space or only for the ROIs (files modified: ephypype/compute_inv_problem.py, ephypype/interfaces/mne/Inverse_solution.py, ephypype/pipelines/fif_to_inv_sol.py

- added function head_to_mni in ephypype/source_space.py

- pep 8 style

### CODE REVIEW

File: ephypype/import_ctf.py [hunk 1]
```diff
@@ -39,174 +39,179 @@ def convert_ds_to_raw_fif(ds_file):
...
+    #subj_path,basename,ext = split_f(ds_file)
 
-	
-	##basename = os.path.splitext(ds_file)[0]
+    ##basename = os.path.splitext(ds_file)[0]
 
-	#print subj_path,basename,ext
-	##0/0
+    # print subj_path,basename,ext
+    # 0/0
```
Comment (jasmainak) [inline ephypype/import_ctf.py]: can you remove these comments?

File: ephypype/import_ctf.py [hunk 2]
```diff
@@ -39,174 +39,179 @@ def convert_ds_to_raw_fif(ds_file):
...
 
-	#raw_fif_file = os.path.join(subj_path,basename + "_raw.fif")
+    #raw_fif_file = os.path.join(subj_path,basename + "_raw.fif")
 
-	#os.system("$MNE_ROOT/bin/mne_ctf2fiff --ds " + os.path.join(subj_path,ds_file) + " --fif " + raw_fif_file)
+    #os.system("$MNE_ROOT/bin/mne_ctf2fiff --ds " + os.path.join(subj_path,ds_file) + " --fif " + raw_fif_file)
 
-	#return raw_fif_file
+    # return raw_fif_file
```
Comment (jasmainak) [inline ephypype/import_ctf.py]: same here. If you have comments that are just code, it should probably be removed

File: ephypype/import_ctf.py [hunk 3]
```diff
@@ -39,174 +39,179 @@ def convert_ds_to_raw_fif(ds_file):
...
-		print(basename)
+        basename = os.path.splitext(ds_f)[0]
 
-		os.system("$MNE_ROOT/bin/mne_ctf2fiff --ds " + os.path.join(subj_path,ds_f) + " --fif " + os.path.join(subj_path,basename + "_raw.fif"))
+        print(basename)
+
+        os.system("$MNE_ROOT/bin/mne_ctf2fiff --ds " + os.path.join(subj_path,
+                                                                    ds_f) + " --fif " + os.path.join(subj_path, basename + "_raw.fif"))
+
+        0 / 0
```
Comment (jasmainak) [inline ephypype/import_ctf.py]: ummm ... why this?

File: ephypype/import_ctf.py [hunk 4]
```diff
@@ -39,174 +39,179 @@ def convert_ds_to_raw_fif(ds_file):
...
-		#0/0
+        print((len(raw.ch_names)))
 
-		print((len(raw.ch_names)))
+        select_electrodes = np.array(
+            [ch_name[0] == 'M' for ch_name in raw.ch_names], dtype='bool')
 
-		select_electrodes = np.array([ch_name[0] == 'M' for ch_name in raw.ch_names],dtype = 'bool')
+        # print select_electrodes
+        #start, stop = raw.time_as_index([0, 100])
```
Comment (jasmainak) [inline ephypype/import_ctf.py]: no comments please!

File: ephypype/import_ctf.py [hunk 5]
```diff
@@ -39,174 +39,179 @@ def convert_ds_to_raw_fif(ds_file):
...
+        basename = os.path.splitext(fif_f)[0]
 
-		basename = os.path.splitext(fif_f)[0]
+        np_filename = os.path.join(subj_path, basename + ".npy")
 
-		np_filename = os.path.join(subj_path,basename + ".npy")
+        np.save(np_filename, electrode_data)
 
-		np.save(np_filename,electrode_data)
+        0 / 0
```
Comment (jasmainak) [inline ephypype/import_ctf.py]: same here, you need to remove debug code

File: ephypype/import_mat.py [hunk 6]
```diff
@@ -167,85 +163,84 @@ def import_mat_to_ts(mat_file,orig_channel_names_file,orig_channel_coords_file):
...
+    raw_data = np.array(mat[field_name], dtype="f")
     print((raw_data.shape))
 
-    #### load electrode names
+    # load electrode names
     elec_names = [line.strip() for line in open(orig_channel_names_file)]
 
     print(elec_names)
-    #0/0
+    # 0/0
```
Comment (jasmainak) [inline ephypype/import_mat.py]: remove

File: ephypype/interfaces/mne/Inverse_solution.py [hunk 7]
```diff
@@ -146,15 +153,24 @@ def _run_interface(self, runtime):
...
+                                     snr, inv_method, parc,
+                                     aseg, aseg_labels, all_src_space,
+                                     ROIs_mean, is_fixed)
+#        self.ts_file, self.labels, self.label_names, self.label_coords = \
+#            compute_ROIs_inv_sol(raw_filename, sbj_id, sbj_dir, fwd_filename,
+#                                 cov_filename, is_epoched, events_id,
+#                                 t_min, t_max, is_evoked,
+#                                 snr, inv_method, parc,
+#                                 aseg, aseg_labels, save_stc=all_src_space,
+#                                 is_fixed=is_fixed)
```
Comment (jasmainak) [inline ephypype/interfaces/mne/Inverse_solution.py]: yikes!

**Comment by annapasca** [comment]
@jasmainak does it fail due to no module h5py?

**Comment by EtienneCmb** [comment]
@annapasca @jasmainak h5py is a dependency?

**Comment by annapasca** [comment]
@EtienneCmb  yes, h5py is a dependency

**Comment by annapasca** [comment]
I close this and create a new PR to neuropycon/master

**Comment by jasmainak** [comment]
`h5py` is in `mne.externals`. Why make another dependency?

**Pull request closed without merging**